src/models/reset.js

```diff
@@ -0,0 +1,32 @@
+// define the Reset model with its content
+const reset = (sequelize, DataTypes) => {
+  const Reset = sequelize.define('reset', {
+    reset_id: {
```

Comment:
I remember mentioning that the id of the table should just be `id`

Reply:
Yes you mentioned it for the users table and i thought it applied only to the users table. There might be conflict referecing users id in other tables if there is no way to tell which id belongs to table 'reset' and which id is from 'users' that was referenced.

Reply:
This is a `reset table` which means  an `id` on this table will represent `reset id`

Reply:
With that, how can i include the user id as a column on the reset table if they both have an id?

Reply:
the `user id` is a `foreign key` on the reset table so it should be named `user_id`.